### ISSUE TITLE
Some mods need to be re-applied after a restart (possibly caused by the preview update KB5040527)

### ISSUE THREAD

**Issue by 001Guy001** [ramensoftware/windhawk#241]
Ones that I've noticed are "Classic Explorer navigation bar", and in "Windows 11 Taskbar Styler" the line I have for the secondary icon on taskbar buttons (Windows.UI.Xaml.Controls.Image#OverlayIcon) stops working until I re-apply the mod.

Not sure if it's just something on my end, but the issues seems to have arrived after I've installed the KB5040527 update (though I've only used Windhawk for a few days before it so I'm not sure if it's the actual cause)

**Comment by 001Guy001** [comment]
Updating that the issue seems to go away after disabling and re-enabling the relevant mods (but not when making some changes to the mods to then re-save them, at least from my brief testing)

**Comment by m417z** [comment]
> "Classic Explorer navigation bar", and in "Windows 11 Taskbar Styler" the line I have for the secondary icon on taskbar buttons (Windows.UI.Xaml.Controls.Image#OverlayIcon) stops working until I re-apply the mod

These are two different mods, and the cause of the issue might be completely different.

> when I disabled every mod and re-enabled the issue didn't come back after a restart

Disabling and re-enabling mods don't change anything, surely not after a restart, so I hardly believe it has something to do with the problem. It's more likely that it just sometimes works and sometimes doesn't.

Please try to come up with a way to reproduce the problem with one of the mods.

**Comment by 001Guy001** [comment]
You are right about the disabling/re-enabling. The issue with "Classic Explorer navigation bar" only appears with the first folder that I open (i.e. explorer launch) after a restart/logon, and then doesn't re-appear when opening more folders (so after it appeared I disabled and re-enabled and thought it fixed it but it's unrelated). Not sure what the steps to reproduce this would be, sorry.

If it's relevant, before Windhawk I've used StartAllBack to revert the explorer bar, and since uninstalled it, so maybe it has some leftover effect if that's possible.

(The issue with Windows 11 Taskbar Styler hasn't returned)
